concent_api/utils/logging.py

```diff
@@ -100,6 +100,20 @@ def log_400_error(endpoint: str, message: Message, client_public_key: str):
     ))
 
 
+def log_no_changes_in_subtask_states(client_public_key: str):
+    logger.info('None of subtask changed state -- CLIENT PUBLIC KEY: {}'.format(
+        client_public_key,
+    ))
+
+
+def log_changes_in_subtask_states(client_public_key: str, count: int, client: str):
+    logger.info('{} subtasks changed their state for client who is {} -- CLIENT PUBLIC KEY: {}'.format(
```

Comment:
When count == 1 it should be `subtask`, but don't focus on this now, I am just leaving it as remark.